api/pkg/auth/service/auth.go

```diff
@@ -152,21 +156,33 @@ func (s *service) HubAuthenticate(res http.ResponseWriter, req *http.Request) {
 		}
 	}
 
-	if err := r.db.Model(&model.User{}).Where("github_login = ?", user.GithubLogin).Update("code", "").Error; err != nil {
+	// Once the user is authenticated clear the code from DB and user struct
+	gitUser.Code = ""
+	if err := r.db.Model(&model.User{}).Where("email = ?", gitUser.Email).Update("code", gitUser.Code).Error; err != nil {
```

Comment:
According to the comment added is `gitUser.Code = ""` clearing the code because in the query you are adding the code